NAS-102084 / 11.3 / Improve jail configuration checks

sonicaj opened the pull request:
This commit improves jail configuration checks and fixes a bug where we referenced a variable which did not exist.

sonicaj committed:
Improve jail configuration checks

This commit improves jail configuration checks and fixes a bug where we referenced a variable which did not exist.

skarekrow commented on iocage_lib/ioc_json.py:
```diff
@@ -1136,20 +1136,17 @@ def check_jail_config(self, conf):
-                        f" '{e.__class__.__name__}:{str(e)}' occured\n"
-                        f"Loading {uuid}'s configuration failed"
+                        f" '{e.__class__.__name__}:{str(e)}' occurred\n"
+                        f"Loading {str(freebsd_version).split('/')[-4]}'s "
```
This would be `"Loading 11.2-RELEASE's configuration failed"`, which is why we used the UUID as it's the jail

sonicaj committed:
Fix release being set inaccurately

This commit fixes an issue where we set release prop of a jail to the release version instead of the jail itself.
